### PR TITLE
Homework two

### PR DESCRIPTION
**Pull request by yurii-shendryk**

### CODE REVIEW

File: src/homework_two/index.ts [hunk 1]
```diff
@@ -0,0 +1,308 @@
...
+
+  get birthYear(): number {
+    return this._birthYear;
+  }
+
+  get grades(): IStudentGrade[] {
+    return this._grades;
+  }
+
+  set grade(newGrade: IStudentGrade) {
```
Comment (KMaxLSVN) [inline src/homework_two/index.ts]: It should be named `grades` to work in pair with getter.

File: src/homework_two/index.ts [hunk 2]
```diff
@@ -0,0 +1,308 @@
...
+    this._grades.push(newGrade);
+  }
+
+  get visits(): IStudentVisit[] {
+    return this._visits;
+  }
+
+  set visit(newVisit: IStudentVisit) {
```
Comment (KMaxLSVN) [inline src/homework_two/index.ts]: The same. It's a mistake in task. You did everything right. But setter should be named `visits`.

File: src/homework_two/index.ts (hunk 1 above)
Comment (yurii-shendryk) [reply src/homework_two/index.ts]: You are right, thx, I'll fix it. Didn't notice that

File: src/homework_two/index.ts (hunk 2 above)
Comment (yurii-shendryk) [reply src/homework_two/index.ts]: I'll fix it, thank you

**Pull request merged**